processing/src/main/java/io/druid/query/GroupByParallelQueryRunner.java

```diff
@@ -56,6 +56,8 @@
 
 public class GroupByParallelQueryRunner<T> implements QueryRunner<T>
```

Comment:
nevermind, that already exists and is a different thing. `GroupByMergedQueryRunner`?